i_rand.cpp

```diff
@@ -44,99 +44,146 @@ rand.c: By Bob Jenkins.  My random number generator, ISAAC.
 #include "i_rand.h"
 #endif
 
-
-#define ind(mm,x)	(*(ub4 *)((ub1 *)(mm) + ((x) & ((RANDSIZ-1)<<2))))
-#define rngstep(mix,a,b,mm,m,m2,r,x) \
-{ \
-	x = *m; \
-	a = (a^(mix)) + *(m2++); \
-	*(m++) = y = ind(mm,x) + a + b; \
-	*(r++) = b = ind(mm,y>>RANDSIZL) + x; \
-}
+#define ind(mm, x) (*(ub4 *)((ub1 *)(mm) + ((x) & ((RANDSIZ - 1) << 2))))
+#define rngstep(mix, a, b, mm, m, m2, r, x)      \
+	{                                            \
+		x = *m;                                  \
+		a = (a ^ (mix)) + *(m2++);               \
+		*(m++) = y = ind(mm, x) + a + b;         \
+		*(r++) = b = ind(mm, y >> RANDSIZL) + x; \
+	}
```

Comment:
This is just VSCode being opinionated about indentation.. no function change here.